handlers/handlers.go

```diff
@@ -110,3 +115,72 @@ func GeographyRender(rend RenderClient) http.HandlerFunc {
 		return
 	}
 }
+
+//GeographyListpageRender ...
+func GeographyListpageRender(rend RenderClient) http.HandlerFunc {
+
+	return func(w http.ResponseWriter, req *http.Request) {
+		var page geographyListPage.Page
+
+		vars := mux.Vars(req)
+		areaTypeID := vars["areaTypeID"]
+
+		resp, err := http.Get(`https://api.dev.cmd.onsdigital.co.uk/v1/code-lists/` + areaTypeID + `/editions/2016/codes`)
+		fmt.Println("error err", err)
+		fmt.Println("error resp.StatusCode", resp.StatusCode)
+		if err != nil {
+			setStatusCode(req, w, err)
+			return
+		}
+		if resp.StatusCode == 404 {
+			fmt.Println("ERROR 404 not 500")
+			setStatusCode(req, w, err) //<--error 500
+			return
+		}
+		b, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			setStatusCode(req, w, err)
+			return
+		}
+
+		var codelistresults models.CodeListResults
+		err = json.Unmarshal(b, &codelistresults)
+		if err != nil {
+			setStatusCode(req, w, err)
+			return
+		}
+		var codelist models.CodeList
+		err = json.Unmarshal(b, &codelist)
+		if err != nil {
+			setStatusCode(req, w, err)
+			return
+		}
+
+		var geographyTypes []geographyListPage.AreaType
+		for i := range codelistresults.Items {
+
+			if i >= 10 {
```

Comment:
guessing this is to limit the page size to 10? need to consider what happens if the code list API returns more than 10 items (e.g. do we support paging, or does that mean they can never get to the 11th item?)

it should also be implemented on the API rather than the frontend, so we can ask for only the 10 we want, rather than asking for all of them and throwing the data away afterwards